### PR TITLE
Update _pipeline_upload.md

### PR DESCRIPTION
**Pull request by nsuma8989**
Adding a note about steps limit in pipeline upload configuration file.

### CODE REVIEW

File: pages/agent/v3/help/_pipeline_upload.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ provided, the command looks for the file in the following locations:
 You can also pipe build pipelines to the command allowing you to create
 scripts that generate dynamic pipelines.
 
+>🚧
+> Configuration file uploaded using pipeline upload command has a limit of 500 steps per file. If there are more than 500 steps per file then it should be split into multiple configuration files with less than 500 steps each and uploaded as separate steps.
```
Comment (mbelton-buildkite) [inline pages/agent/v3/help/_pipeline_upload.md:39]: It looks like this file is autogenerated, so any edits here will be overridden:

> This file is auto-generated by scripts/update-agent-help.sh, please update the
agent CLI help in https://github.com/buildkite/agent and run the generation
script.

**Review by mbelton-buildkite — COMMENTED** [review]
This page is generated from the agent's CLI docs, so you'll need to:

1. Make the change in the Agent repo.
   1. Update [this file](https://github.com/buildkite/agent/blob/main/clicommand/pipeline_upload.go#L46).
   2. I suggest using this wording:
       > The configuration file has a limit of 500 steps per file. Configuration files with over 500 steps must be split into multiple files and uploaded in separate steps.
   1. PR the agent change.
1. Run the `scripts/update-agent-help.sh` script in this repo to generate the change in the docs.
1. PR the docs change.

Let me know if you need help 😅

File: pages/agent/v3/help/_pipeline_upload.md (hunk 1 above)
Comment (nsuma8989) [reply pages/agent/v3/help/_pipeline_upload.md:39]: @mbelton-buildkite I created PR https://github.com/buildkite/agent/pull/2046. Once that is approved I can update this PR running the script.

**Comment by mbelton-buildkite** [comment]
Nice work @nsuma8989! I see this change included in the [latest agent docs](https://github.com/buildkite/docs/pull/2157/files#diff-f4c1c6c19f1ee290e48eb4d3d9e41e74db01a7f821570e1fe9b098a74761a9b6R36). Shall we just close this one now?

**Comment by nsuma8989** [comment]
> Nice work @nsuma8989! I see this change included in the [latest agent docs](https://github.com/buildkite/docs/pull/2157/files#diff-f4c1c6c19f1ee290e48eb4d3d9e41e74db01a7f821570e1fe9b098a74761a9b6R36). Shall we just close this one now?

Yes @mbelton-buildkite Closing this one as change is live in docs now.

**Pull request closed without merging**